includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -690,7 +690,6 @@ public function prepare_source( $user_id, $force_save_source = false ) {
 		}
 
 		return (object) array(
-			'is_intent'     => false,
```

Comment:
This was added as part of the SCA work back in a1d897a1527f14f2a799283301ccb72aaacc775b. I haven't followed through to exactly what it's being used for, but do we want to remove it here?

Reply:
This removal is not legacy-modal related, but while I was doing cleanup I found out that this flag was still in place.

It was supposed to be used when we were supporting non-PI and PI-based credit card payments within the gateway and the reason for that actually used to be Stripe Checkout. Now everything is PI-based, so the flag/prop is not needed anymore.